### PR TITLE
removed DIB_ELREPO_PKGLIST

### PR DESCRIPTION
**Pull request by g0rgamesh**
as installing packages can be done via general packages section
removed hard tabs

### CODE REVIEW

File: elements/elrepo/pre-install.d/01-elrepo [hunk 1]
```diff
@@ -8,20 +8,15 @@ set -euo pipefail
 
 case "$DISTRO_NAME" in
     centos* | rhel* | rocky*)
-	dnf install -y elrepo-release.noarch
```
Comment (markgoddard) [inline elements/elrepo/pre-install.d/01-elrepo:11]: Old indentation made more sense to me

**Review by markgoddard — CHANGES_REQUESTED** [review]
Won't this break existing users of the element?

File: elements/elrepo/pre-install.d/01-elrepo (hunk 1 above)
Comment (g0rgamesh) [reply elements/elrepo/pre-install.d/01-elrepo:11]: That might be the case. I'll just drop it then.

**Comment by g0rgamesh** [comment]
> Won't this break existing users of the element?

**Comment by g0rgamesh** [comment]
abandoning, change might break usage for old users

**Pull request closed without merging**